extensions/BugModal/web/bug_modal.css

```
@@ -729,6 +729,15 @@ div.comment-text blockquote {
     padding: 0.5em 10px;
 }
 
+
+div.comment-text blockquote p:first-child {
```

Agree with @purelogiq. It would be:

```suggestion
div.comment-text blockquote > :first-child {
```